### ISSUE TITLE
install / run configuration discrepancy 

### ISSUE THREAD

**Issue by pettai** [log2timeline/dftimewolf#23]
`root@plaso:~# cd dftimewolf-master/`
`root@plaso:~/dftimewolf-master# pip install .`

```
[...]
    byte-compiling /usr/local/lib/python2.7/dist-packages/dftimewolf/cli/dftimewolf_recipes.py to dftimewolf_recipes.pyc
    running install_data
    creating /usr/local/dftimewolf
    error: can't copy 'dftimewolf/dftimewolf.json': doesn't exist or not a regular file

    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-nw3biE-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-7n3gw2-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-nw3biE-build/
```

`root@plaso:~/dftimewolf-master# ls dftimewolf/`

```
cli/         config.json  config.py    __init__.py  lib/
```

This (https://github.com/log2timeline/dftimewolf/blob/master/setup.py#L32) should probably be corrected to config.json

alternatively

config.json should be renamed to dftimewolf.json (as per
https://github.com/log2timeline/dftimewolf/blob/master/dftimewolf/cli/dftimewolf_recipes.py#L54)

**Comment by pettai** [comment]
Suggested fix: https://github.com/log2timeline/dftimewolf/pull/26

**Comment by tomchop** [comment]
Just merged #26. Closing this out. Thanks for your contribution!
